third_party/Makefile

```diff
@@ -0,0 +1,39 @@
+go-oniguruma:
+	DIR=$$(mktemp -d) && \
+	echo "$(abspath $@)" && \
+	cd $$DIR && \
+	git clone --branch v1.2.1 git@github.com:go-enry/go-oniguruma.git go-oniguruma && \
+	cd go-oniguruma && \
+	git apply $(abspath go-oniguruma.patch) && \
+	rm -rf "$(abspath $@)" && \
+	cp -a $$DIR"/go-oniguruma" "$(abspath $@)"
+
+oniguruma:
+	DIR=$$(mktemp -d) && \
+	echo "$(abspath $@)" && \
+	cd $$DIR && \
+	git clone --branch v6.9.10 git@github.com:kkos/oniguruma.git oniguruma && \
+	cd oniguruma && \
+	git apply $(abspath oniguruma.patch) && \
+	rm -rf "$(abspath $@)" && \
+	cp -a $$DIR"/oniguruma" "$(abspath $@)"
+
+.PHONY: update_go-oniguruma.patch
+update_go-oniguruma.patch:
```

Comment:
Why did you call this `update_go-oniguruma.patch` instead of just calling it `go-oniguruma.patch` since that's what it creates? These aren't phony rules; they're actually rules to make a file.